tools/generate_summary.py

```
@@ -0,0 +1,122 @@
+                for line in parts[1].splitlines():
+                    if line.strip().startswith('title:'):
+                        title_val = line.split('title:', 1)[1].strip()
+                        return title_val.strip('"').strip("'")
+
+        # Check H1 header
+        for line in content.splitlines():
+            line = line.strip()
+            if line.startswith('# '):
+                return line[2:].strip()
+    except Exception:
+        pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)